reproducible/reference_binaries_ubuntu-18.04.sha256sum

```diff
@@ -1,9 +1,9 @@
 3feb5d29a3d669107b460a00391440be4ebc5e50461f9ef3248714f4f99c070e	third_party/tock/target/thumbv7em-none-eabi/release/nrf52840dk.bin
-3c4dde09e8e6082a15bbf9d9eedd12de63f70d85c240292aea246f2d83c54f07	target/nrf52840dk_merged.hex
```

Comment:
I'm not sure I understand why adding some fuzzing testing would have some impact on the hash of the binary of the firmware.
Am I missing something?

Reply:
Could it be that we merged with somethign that changed the hashes?